### PR TITLE
Refactor nav (w/o search) to align with PSD; fix submenu bug

### PR DESCRIPTION
**Pull request by JoeShep**
Getting navigation in a demo-ready state before merging in search. Updates the positioning and sizing of the nav items and logo to match the three breakpoint designs.

Also fixes the bug that hid the submenu below the nav div.
screen 20shot 20on 202015-04-08 20at 2018 3a34 3a36
screen shot 2015-04-08 at 8 56 20 pm

### CODE REVIEW

File: ui/components/navigation/_navigation.scss [hunk 1]
```diff
@@ -10,61 +13,81 @@ $mobile-breakpoint: 401;
...
+    margin: 0 auto;
+    border-bottom: 1px solid rgba(255,255,255,.3);
+  }
+
+  @include respond-to($wide-breakpoint) {
+    height: $nav-height--wide;
   }
 }
 
 .logo--header {
```
Comment (jcreamer898) [inline ui/components/navigation/_navigation.scss]: Should this be `.logo__header`?

File: ui/components/navigation/_navigation.scss [hunk 2]
```diff
@@ -114,29 +136,29 @@ $mobile-breakpoint: 401;
     .navigation-wrapper,
     .logo--header,
     {
-      -webkit-transform:  translateX(-$responsive-nav-width);
-      -moz-transform:  translateX(-$responsive-nav-width);
-      -ms-transform:  translateX(-$responsive-nav-width);
-      -o-transform:  translateX(-$responsive-nav-width);
-      transform:  translateX(-$responsive-nav-width);
+      -webkit-transform: translateX(-$responsive-nav-width);
```
Comment (jcreamer898) [inline ui/components/navigation/_navigation.scss]: Not sure we need all these different vendor prefixes in here. We have auto prefecture for that?

File: ui/components/navigation/_hamburger.scss [hunk 3]
```diff
@@ -34,40 +32,37 @@
...
 }
 .is-open .hamburger-menu__svg__toggle__bar:nth-of-type(2) {
   opacity: 0;
 }
 .is-open .hamburger-menu__svg__toggle__bar:nth-of-type(3) {
   -webkit-transform: rotate(45deg) translateY(0em) translateX(0em);
-  transform: rotate(45deg) translateY(0em) translateX(0em);
+          transform: rotate(45deg) translateY(0em) translateX(0em);
 }
-
```
Comment (jcreamer898) [inline ui/components/navigation/_hamburger.scss]: LOTS of vendor prefixes here, again, autoprefixer?

File: ui/components/navigation/_navigation.scss (hunk 1 above)
Comment (stevebrownlee) [inline ui/components/navigation/_navigation.scss]: It should be either `logo-header` if there is only one state of the logo, or the current `logo--header` if your intent is to signify that the logo is rendered in more than one state.

File: ui/components/navigation/_navigation.scss (hunk 2 above)
Comment (stevebrownlee) [inline ui/components/navigation/_navigation.scss]: :+1:

**Pull request merged**